src/ui/backend/canvas/ViewBacking.js

```diff
@@ -169,19 +195,17 @@ var ViewBacking = exports = Class(BaseBacking, function () {
 			this._view.render && this._view.render(ctx, opts);
```

Comment:
The try-catch block here could be moved into the conditional to protect broken render fns:
```
if (this._view.render) {
  try { this._view.render(ctx, opts); } catch(e) {}
}
```